lib/tasks/users.rake

```diff
@@ -0,0 +1,27 @@
+namespace :users do
+  desc "Update all trial and editor users to be standard users - dry run"
+  task update_user_roles_to_standard_dry_run: :environment do
+    ActiveRecord::Base.transaction do
+      update_user_roles
+      Rails.logger.info "users:update_user_roles_to_standard_dry_run rollback"
+      raise ActiveRecord::Rollback
+    end
+  end
+
+  desc "Update all trial and editor users to be standard users"
+  task update_user_roles_to_standard: :environment do
+    update_user_roles
+  end
+end
+
+def update_user_roles
+  Rails.logger.info("Number of trial users: #{User.where(role: 'trial').count}")
+  Rails.logger.info("Number of editor users: #{User.where(role: 'editor').count}")
+  Rails.logger.info("Number of standard users: #{User.where(role: 'standard').count}")
+
+  User.where(role: %w[trial editor]).update_all(role: "standard")
+
+  Rails.logger.info("Number of trial users: #{User.where(role: 'trial').count}")
+  Rails.logger.info("Number of editor users: #{User.where(role: 'editor').count}")
+  Rails.logger.info("Number of standard users: #{User.where(role: 'standard').count}")
+end
```

Comment:
small thing might not be needed. Is it helpful to log super admin and org admin counts as well just to be sure they haven't changed?

Reply:
Good suggestion. I've added that 👍